docs/source/teams/index.rst

```diff
@@ -75,9 +75,9 @@ pages on this site apply to Teams deployments as well.
     :button_link: cloud_media.html
 
 .. customcalloutitem::
-    :header: Roles and permissions
-    :description: Learn how to configure roles and access permissions for users of your deployment.
-    :button_text: Learn more about permissions
+    :header: Roles, groups and permissions
+    :description: Learn how to configure roles, groups and access permissions for users of your deployment.
+    :button_text: Learn more about user groups and permissions
```

Comment:
nit: suggest leaving `button_text` unchanged so the button isn't too wide (messes with formatting)